### PR TITLE
Add copymanga source

### PR DESCRIPTION
**Pull request by sunbeams001**
#3795

### CODE REVIEW

File: src/zh/copymanga/src/eu/kanade/tachiyomi/extension/zh/copymanga/CopyManga.kt [hunk 1]
```diff
@@ -0,0 +1,323 @@
...
+        throw UnsupportedOperationException("This method should not be called!")
+
+    // NOT WORK!!! As for now, copymanga has different logic in polular and search page
+    override fun getFilterList() = FilterList(
+        ThemeFilter(
+            "题材",
+            "theme",
+            arrayOf(
+                Pair("全部", "")
+//                Pair("愛情", "aiqing"),
```
Comment (CarlosEsco) [inline src/zh/copymanga/src/eu/kanade/tachiyomi/extension/zh/copymanga/CopyManga.kt:188]: why are these all commented out?

Comment (sunbeams001) [reply src/zh/copymanga/src/eu/kanade/tachiyomi/extension/zh/copymanga/CopyManga.kt:188]: // NOT WORK!!! As for now, copymanga has different logic in polular and search page
As I commented in the code, for now the framework flow will call "searchManga" progress in polupar page when consider filters, but in this site, it's two different implementation, the popular page by parse html, the search page by api, and only the popular page can filter comics.
I don't know if the site will change the search logic in the future, so I remain these filter items for now.

Comment (CarlosEsco) [reply src/zh/copymanga/src/eu/kanade/tachiyomi/extension/zh/copymanga/CopyManga.kt:188]: I would remove them then if they are not used at this time.

Comment (sunbeams001) [reply src/zh/copymanga/src/eu/kanade/tachiyomi/extension/zh/copymanga/CopyManga.kt:188]: OK, got it.

**Pull request merged**